pyigm/metallicity/mcmc.py

```diff
@@ -459,17 +490,26 @@ def plotinfo(self, sampler, use_pkl=False):
         ax.set_xticklabels(xaxislabels)
         ax.tick_params(axis='both', direction='in', top=True)
         ax.set_xlabel('Steps')
-        fig.text(0.50,0.93,self.info['name']+", UVB="+self.UVB+", logUconstraint="+str(self.logUconstraint),horizontalalignment='center',fontsize=8)
+        # fig.text(0.5,0.93,plot_title,horizontalalignment='center',fontsize=10)
         fig.savefig(self.outsave+'/'+self.info['name']+'_chains.pdf')
+        plt.close(fig)
 
-        #now do a corner plot
+
+        ######
+        ##Corner Plot
+        ######
+        
         samples = sampler.chain[:,self.burn:, :].reshape((-1,self.ndim))
-        cfig = corner.corner(samples, labels=self.mod_axistag, quantiles=[0.05,0.5,0.95],verbose=False)
-        cfig.text(0.33,0.90,self.info['name']+", UVB="+self.UVB+", logUconstraint="+str(self.logUconstraint),horizontalalignment='left',fontsize=10)
+        cfig = corner.corner(samples, labels=self.mod_axistag, label_kwargs = {"fontsize": 16}, quantiles=[0.05,0.5,0.95],verbose=False)
```

Comment:
Is this for an updated corner method?

Reply:
No, this is the same plot.  I'm just adding (1) a plot title (to match the other plots) and (2) adjusting font size of the axis titles (x and y labels).  I now have the plot title as a variable, too, so it's defined in a single place.